test/malli/core_test.cljc

```diff
@@ -586,6 +586,74 @@
                                         [false [:sequential int?] #{1 2 3}]
                                         [false [:sequential int?] nil]]
 
+                          "cat" [; same as [:cat {:into nil} ...]
+                                 [true [:cat [:x int?]] '(1)]
+                                 [true [:cat [:x int?]] [1]]
+
+                                 ; aka :cat*
+                                 [true [:cat {:into :any} [:x int?]] '(1)]
```

Comment:
does `:into` mean just type validation? If so, the name is bit misleading.

Reply:
It is meant for validation and generation. I took this wording from an option of https://clojure.org/guides/spec#_collections and would not mind changing it.

Reply:
@ikitommi Do you have a suggestion for an alternative to `:into`?